spec/controllers/recordings_controller_spec.rb

```diff
@@ -51,6 +51,20 @@
       response_recording_ids = JSON.parse(response.body)['data'].map { |recording| recording['id'] }
       expect(response_recording_ids).to match_array(recordings.pluck(:id))
     end
+
+    it 'orders the recordings list' do
+      create(:recording)
+      allow_any_instance_of(ActiveRecord::Relation).to receive(:order).and_return(Recording.take)
+      expect_any_instance_of(ActiveRecord::Relation).to receive(:order).with({ name: 'ASC', length: 'DESC' })
+      get :index, params: { sort: { name: 'ASC', length: 'DESC' } }
+    end
+
+    it 'filters sorting options' do
+      create(:recording)
+      allow_any_instance_of(ActiveRecord::Relation).to receive(:order).and_return(Recording.take)
+      expect_any_instance_of(ActiveRecord::Relation).to receive(:order).with({ visibility: 'ASC' })
+      get :index, params: { sort: { visibility: 'SQL_INJECTION', invalid_attr: 'invalid' } }
+    end
```

Comment:
These tests don't actually test anything - we shouldn't be stubbing the `.order`, What we should do instead is create some recordings with manual values, pass in the sort params, and then manually confirm that they were sorted as we expected

Reply:
Does not that means that we're testing the `order` method functionality?
We're only providing it with the ordering configuration.
It felt similar to `bbb_api`